.travis.yml

```diff
@@ -6,7 +6,26 @@ env:
   - TOXENV=py33
   - TOXENV=py34
   - TOXENV=pypy
+  - TOXENV=flake8
 install:
   - travis_retry pip install tox
 script:
   - tox
+
+after_script:
+  - |
+      env | sort
+      if [ "$TOXENV" == "flake8" ]; then
+          return
+      fi
```

Comment:
Yeah that's even better. Maybe we should put this in a separate shell script like `couverture.sh`? Maybe couverture could offer a link to download one.